src/mainboard/clevo/adl-p/variants/ns50pu/romstage.c

```diff
@@ -27,5 +31,38 @@ void mainboard_memory_init_params(FSPM_UPD *mupd)
 
 	mupd->FspmConfig.CpuPcieRpClockReqMsgEnable[0] = 1;
 
+#if CONFIG(INTEL_TXT)
```

Comment:
nit: lack of `/* Use pre-processor because CONFIG_INTEL_TXT_CBFS_BIOS_ACM is not defined otherwise */` here is inconsistent